frontend/src/App.tsx

```diff
@@ -481,83 +460,71 @@ export default function App() {
   }, []);
 
   return (
-    <>
-      <ToastContainer />
-      <Routes>
-        <Route
-          path="/"
-          element={
-            <LobbyPage
-              appState={state}
-              isLargeScreen={isLargeScreen}
-              handleToggleTheme={handleToggleTheme}
-              handleToggleDarkMode={handleToggleDarkMode}
-              handlePlayerName={handlePlayerName}
-              handleEloId={handleEloId}
-              handleToken={handleToken}
-              handleIsPrivate={handleIsPrivate}
-              handleNumRows={handleNumRows}
-              handleNumCols={handleNumCols}
-              handlePosSetting={handlePosSetting}
-              handleJoinCode={handleJoinCode}
-              handleCreateGame={handleCreateGame}
-              handleJoinGame={handleJoinGame}
-              handleAcceptChallenge={handleAcceptChallenge}
-              handleReturnToGame={handleReturnToGame}
-              handleViewGame={handleViewGame}
-              handleLocalGame={handleLocalGame}
-              handleComputerGame={handleComputerGame}
-              handleSolvePuzzle={handleSolvePuzzle}
-              handleHasOngoingGameInServer={handleHasOngoingGameInServer}
-            />
-          }
-        />
-        <Route
-          path="/game/:gameId"
-          element={
-            <GamePage
-              clientParams={state}
-              isLargeScreen={isLargeScreen}
-              handleReturnToLobby={handleReturnToLobby}
-              handleToggleDarkMode={handleToggleDarkMode}
-              handleToggleTheme={handleToggleTheme}
-              wasmAIGetMove={wasmAIGetMove}
-            />
-          }
-        />
-        <Route
-          path="/puzzle/:puzzleId"
-          element={
-            <GamePage
-              clientParams={state}
-              isLargeScreen={isLargeScreen}
-              handleReturnToLobby={handleReturnToLobby}
-              handleToggleDarkMode={handleToggleDarkMode}
-              handleToggleTheme={handleToggleTheme}
-            />
-          }
-        />
-        <Route path="*" element={<ErrorPage />} />
-      </Routes>
-    </>
+    <Routes>
```

Comment:
It looks like you might have accidentally removed the `<ToastContainer \>`?
I'm guessing you do not see any toasts? Can you re-add it?